src/assets/styles/_shadow.scss

```diff
@@ -1,4 +1,4 @@
-@mixin componentShadowVars() {
+%shadow-vars {
```

Comment:
What does the % symbol mean here?

Reply:
It makes this a Sass placeholder.
You can see its usage in `global.scss`.

Reply:
Here we go: https://sass-lang.com/documentation/style-rules/placeholder-selector definitely something I haven't used before.  Thanks!